mongodbatlas/provider.go

```diff
@@ -39,6 +39,7 @@ var (
 
 const (
 	endPointSTSDefault = "https://sts.amazonaws.com"
+	DeprecationMessage = "this resource is deprecated, and will be removed in %s release, please transition as soon as possible to %s"
```

Comment:
Hey @andreaangiolillo, this is very clear, some suggestions to make it more concise: 
```suggestion
	DeprecationMessage = "this resource is deprecated and will be removed in %s, please transition to %s"
```